src/main/java/laddergame/domain/Participants.java

```diff
@@ -0,0 +1,36 @@
+package laddergame.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Participants {
+    private static final int MIN_PARTICIPANTS_SIZE = 2;
+    private static final String PARTICIPANTS_NULL_EXCEPTION = "참여자 이름 목록은 null이 될 수 없습니다.";
+    private static final String PARTICIPANTS_MIN_SIZE_EXCEPTION = "참여자 이름 목록은 2명 미만이 될 수 없습니다.";
+
+    private final List<Name> names;
```

Comment:
names를 일급 컬렉션으로 추상화해보는건 어떨까요?

Reply:
수정했습니다 ! 😁

```suggestion
    private final Names names;
```

일급 컬렉션으로 추상화해서 Names에 이름 목록이 2명 미만 검증을 넘겼습니다. 


```java
public class Names {
  private static final int MIN_PARTICIPANTS_SIZE = 2;
  private static final String NAMES_NULL_EXCEPTION = "Name 리스트는 null일 수 없습니다.";
  private static final String PARTICIPANTS_MIN_SIZE_EXCEPTION = "Names 리스트는 2명 미만이 될 수 없습니다.";

  private final List<Name> names;

  public Names(final List<Name> names) {
      validateNames(names);
      this.names = names;
  }

  // ...
```